ccm_web/package.json

```diff
@@ -24,9 +33,14 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.2.0",
```

Comment:
NestJS is the proposed framework for the NodeJS backend. React is the framework we are using in the frontend (see `client`). Because it's all JavaScript, we can use a single `package.json` to collect dependencies for both the server and client. I think there is some debate about whether to use one or multiple `package.json` files, but this is where we're at for now.

Reply:
@lsloan, if you want to put together a list of questions about how all the pieces fit together, I am happy to meet with you individually to explain/discuss.